### PR TITLE
Adds Object.get_or_upload_file

### PR DESCRIPTION
**Pull request by davecap**

### CODE REVIEW

**Comment by jsh2134** [comment]
does this need a doc or something?

```Error in Object.get_or_upload_file(*******) :
  could not find function "Object.get_or_upload_file"
```

**Comment by davecap** [comment]
Yep needed a `make clean` then `make` for some reason.

File: R/object.R [hunk 1]
```diff
@@ -328,3 +328,38 @@ Object.upload_file <- function(local_path, vault_id, vault_path, filename, env =
...
+#' @examples \dontrun{
+#' Object.get_or_upload_file("my_file.json.gz", vault$id, "/parent/directory/")
+#' }
+#'
+#' @references
+#' \url{https://docs.solvebio.com/}
+#'
+#' @export
+Object.get_or_upload_file <- function(local_path, vault_id, vault_path, filename, env = solvebio:::.solveEnv) {
+    object_full_path = paste(vault$full_path, path, filename, sep="/")
```
Comment (jsh2134) [inline R/object.R]: `vault` is not defined here

**Comment by davecap** [comment]
Good catch, updated the function a bit. May depend on an API fix for handling double slashes in paths.

**Pull request merged**